### ISSUE TITLE
Save SCC generated by OpenJ9 on test failure

### ISSUE THREAD

**Issue by dsouzai** [AdoptOpenJDK/openjdk-tests#861]
For any Shared Classes tests for OpenJ9, as well as when Shared Classes becomes default in Openj9, it would be very helpful for debugging for the test harness to save the SCC file generated or used by the test. I believe the test harness already saves the core files, so this would be an extension of that.

**Comment by smlambert** [comment]
@llxia - is this something that you would want to archive to test jobs in the event of test failure?

**Comment by llxia** [comment]
Yes, we should archive it if we haven't already done so. Move this issue to TKG repo as the test framework should set the SCC path.
